library/src/webapp/js/portal/portal.sidebar.sites.js

```diff
@@ -92,6 +89,32 @@ class SitesSidebar {
     this._element.classList.remove("d-none");
   }
 
+  /**
+   * Switches the attributes on the cloned pinned or unpinned site element
+   */
+  _updatePinned(clone, from, to) {
```

Comment:
It would be beneficial if we could add more detailed comments to clarify the parameters `(from, to)`. I think it would be helpful.